monkey/infection_monkey/telemetry/i_telem.py

```diff
@@ -0,0 +1,29 @@
+import abc
+
+
+class ITelem(metaclass=abc.ABCMeta):
```

Comment:
This naming convention is used nowhere in the current codebase and it shouldn't be, since there's no interfaces in python. Maybe `AbstractTelem` is better?

Reply:
Python has no interfaces because you're supposed to use [duck typing](https://realpython.com/lessons/duck-typing/). But if you're going to use type hinting, then cases will arise when you need interfaces.

In this case, I want to establish an interface for telemetry. Interfaces contain no implementation, so you can implement multiple interfaces without being concerned with [the pitfalls of multiple inheritance](https://stackoverflow.com/questions/406081/why-should-i-avoid-multiple-inheritance-in-c)

This is named with an "I" because we don't want any implementation to creep into it. If we name it "AbstractTelem", that would indicate we're ok with some of its methods containing implementations, but we have BaseTelem for that.